### DISCUSSION TITLE
Calendar month view - possible?

### DISCUSSION THREAD

**Discussion by Serhioromano** [tailwindlabs/tailwindcss#3337] [Help]
I know I can create 7x5 grid. That part works perfect. But can anyone in a playground show me how to create event that would be as line though different cells?

![2021-01-12_19-14-10](https://user-images.githubusercontent.com/650741/104319002-6ce65380-550a-11eb-899b-9b9e1ebd8d97.png)

Would it be possible to do with tailwind?

**Comment by adamwathan** [comment]
Here's a quick idea I whipped up in 3-4 minutes, not sure if this approach will work for your needs but might be a helpful start:

https://play.tailwindcss.com/AIl5AxxVjU

**Reply by Serhioromano** [reply]
Thank you very much! I have popular calendar web component done with bootstrap 3 and I want to change it to vanilla JS and Tailwind.

If you do not mind, I ask another questions. 

1. I did not manage to make its height 100%
2. Is grid the way to go or better make flex-col for 5 rows and then each row makes 7 cells? What way would you go?
